### PR TITLE
Fixed displaying available quantity on sidebar

### PR DESCRIPTION
**Pull request by SebaKas21**

### CODE REVIEW

File: fe/src/project/AspNetCore/shared/components/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -130,12 +130,12 @@ const Sidebar = (props) => {
                                                         }
                                                     </a>
                                                     <div className="stock-details">
-                                                        {carouselItem.availableQuantity && carouselItem.availableQuantity > 0 &&
+                                                        {carouselItem.availableQuantity > 0 && carouselItem.availableQuantity &&
```
Comment (dawiddworak88) [inline fe/src/project/AspNetCore/shared/components/Sidebar/Sidebar.js]: **suggestion (to check):** looks redundant

**Review by dawiddworak88 — CHANGES_REQUESTED** [review]

**Comment by dawiddworak88** [comment]
Hi @SebaKas21, would you check it and make it as a hotfix? I'd deploy it first today.

**Comment by SebaKas21** [comment]
@dawiddworak88 

Okay

**Comment by SebaKas21** [comment]
i created branch hotfix so i close PR

**Pull request closed without merging**